Gemfile

```diff
@@ -66,6 +66,7 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem 'bullet'
```

Comment:
- [ ] @Team I can see you have gem bullet installed in your app for this milestone, The project requires us not to use this gem but instead use this [article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh)
![image](https://github.com/danielsafari143/blog-app/assets/61408860/73ba6222-76b3-46fe-8142-d3a3328c0ca5)